frontend/src/modules/app/layouts/LandingPageRoute.tsx

```diff
@@ -1,7 +1,6 @@
 /**
- * This is the doc comment for Landing Page Layout
- * @packageDocumentation
- * @module landing-page-layout
+ * @summary Layout file for landing page
```

Comment:
Where should we go to supply the main content for the landing page?

Reply:
Main content for all the files are provided in `Routes.tsx` file, following is the example of landing page
```
<LandingPageRoute component={LandingPage} />
```